students/david_russo/lesson2/drawing_grids.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# This script draws grids in an increasingly general and abstract manner 
+
+# Part 1
+def print_fixed_grid():
+	plus = '+ '
+	minus = '- ' 
+	# define distinct strings
+	full_line = '+ ' * 1 + '- ' * 4 + '+ ' + '- ' * 4 + '+'
+	sparse_line = '| ' + '  ' * 4 + '| ' + '  ' * 4 + '|'
```

Comment:
You defined `plus` and `minus` above but didn't use them in the `full_line` variable.  I also might suggest creating variables `bar = '| '` and `space = ' '` and use to the same effect.

Reply:
You could even define them at the top outside the functions so you can use in both functions as global variables.